### PR TITLE
refactor: small improvement in `loadInNamespace` Autoloader

### PR DESCRIPTION
**Pull request by ddevsr**
**Description**
Move function call in nested loop to outside.

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Autoloader/Autoloader.php [hunk 1]
```diff
@@ -280,11 +280,14 @@ protected function loadInNamespace(string $class)
         }
 
         foreach ($this->prefixes as $namespace => $directories) {
+            $classNamespaceFind = strpos($class, $namespace);
+            $fileNamespace      = str_replace('\\', DIRECTORY_SEPARATOR, substr($class, strlen($namespace)));
```
Comment (kenjis) [inline system/Autoloader/Autoloader.php:284]: Thank you for refactoring. The intent is good. But the variables are a bit not good.

If you see `$classNamespaceFind` or `$fileNamespace`, what do you think the variable represents?

I guess `$classNamespaceFind` is true when the class namespace is found.
Also I guess `$fileNamespace` means the namespace of the file.

**Comment by kenjis** [comment]
I'm not sure, but the code should be like this?
```php
        foreach ($this->prefixes as $namespace => $directories) {
            if (strpos($class, $namespace) === 0) {
                $relativeClassPath = str_replace('\\', DIRECTORY_SEPARATOR, substr($class, strlen($namespace)));

                foreach ($directories as $directory) {
                    $directory = rtrim($directory, '\\/');

                    $filePath = $directory . $relativeClassPath . '.php';
                    $filename = $this->includeFile($filePath);

                    if ($filename) {
                        return $filename;
                    }
                }
            }
        }
```

**Comment by ddevsr** [comment]
> I'm not sure, but the code should be like this?
> 
> ```
>         foreach ($this->prefixes as $namespace => $directories) {
>             if (strpos($class, $namespace) === 0) {
>                 $relativeClassPath = str_replace('\\', DIRECTORY_SEPARATOR, substr($class, strlen($namespace)));
> 
>                 foreach ($directories as $directory) {
>                     $directory = rtrim($directory, '\\/');
> 
>                     $filePath = $directory . $relativeClassPath . '.php';
>                     $filename = $this->includeFile($filePath);
> 
>                     if ($filename) {
>                         return $filename;
>                     }
>                 }
>             }
>         }
> ```

Thankyou suggest, l like that cut unused loop directory if not exists class namespace

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
@ddevsr Thank you!

**Pull request merged**